burnman/anisotropy.py

```diff
@@ -0,0 +1,449 @@
+            cijs.insert(4, cij[3]) # C24 = -C14
+            cijs.insert(6, cij[5]) # C25 = -C15
+            index_lists = [[(0, 0), (1, 1)], # C11
+                           [(0, 1)], # C12
+                           [(0, 2), (1, 2)], # C13
+                           [(0, 3), (4, 5)], # C14
+                           [(1, 3)], # C24
+                           [(0, 4)], # C15
+                           [(1, 4), (3, 5)], # C25
+                           [(2, 2)], # C33
+                           [(3, 3), (4, 4)], # C44
+                           [(5, 5)]] # C66
+            
+        elif crystal_system=='orthorhombic':
+            assert len(cijs) == 9
+            index_lists = [[(0, 0)], # C11
+                           [(0, 1)], # C12
+                           [(0, 2)], # C13
+                           [(1, 1)], # C22
+                           [(1, 2)], # C23
+                           [(2, 2)], # C33
+                           [(3, 3)], # C44
+                           [(4, 4)], # C55
+                           [(5, 5)]] # C66
+            
+        elif crystal_system=='monoclinic':
+            assert len(cijs) == 13
+            index_lists = [[(0, 0)], # C11
+                           [(0, 1)], # C12
+                           [(0, 2)], # C13
+                           [(0, 4)], # C15
+                           [(1, 1)], # C22
+                           [(1, 2)], # C23
+                           [(1, 4)], # C25
+                           [(2, 2)], # C33
+                           [(2, 4)], # C35
+                           [(3, 3)], # C44
+                           [(3, 5)], # C46
+                           [(4, 4)], # C55
+                           [(5, 5)]] # C66
+            
+        elif crystal_system=='triclinic':
+            assert len(cijs) == 21
+            index_lists=[[(i, j)] for i in range(6) for j in range(i, 6)]
+            
+        else:
+            raise Exception('Crystal system not recognised. Must be one of: '
+                            'isotropic, cubic, hexagonal, tetragonal I, '
+                            'tetragonal II, rhombohedral I, rhombohedral II, '
+                            'orthorhombic, monoclinic or triclinic.')    
+            
+        
+        C = np.zeros([6, 6])
+        for i, index_list in enumerate(index_lists):
+            for indices in index_list:
+                C[indices] = cijs[i]
+                C[indices[::-1]] = cijs[i]
+        return C
+        
+    def _voigt_index_to_ij(self, m):
+        """
+        Returns the ij (or kl) indices of the 
+        stiffness tensor which correspond to those 
+        of the Voigt notation m (or n).
+        """
+        if m == 3:
+            i=1
+            j=2
+        elif m == 4:
+            i=0
+            j=2
+        elif m == 5:
+            i=0
+            j=1
+        else:
+            i=m
+            j=m
+        return i, j
+
+    def _normalize(self, vector):
```

Comment:
Done. I needed to restructure tools.py to avoid circular imports, but I think it's better now anyway.